reactor-core/src/main/java/reactor/core/scheduler/ParallelScheduler.java

```diff
@@ -89,6 +89,27 @@ public boolean isDisposed() {
 		return current != null && current.currentResource == SHUTDOWN;
 	}
 
+	@Override
+	public void init() {
+		SchedulerState<ScheduledExecutorService[]> b =
```

Comment:
lets add as the very first line in the method the following to ensure we are not initing when already inited:

```java
if (a != null) {
  if (a.currentResource == SHUTDOWN) {
    throw ...
  }
  return;
}
```

This pattern is called [test and test-and-set](https://en.wikipedia.org/wiki/Test_and_test-and-set)

Reply:
Thanks.